src/redux/modules/mdb.js

```diff
@@ -395,11 +95,9 @@ const normalizeLinkedUnits = (linkedUnits, type, parentId, state) => {
   return { ids, updatedCuById };
 };
 
-const onReceiveCollections = (state, action) => {
-  const items = action.payload || [];
-
+const onReceiveCollections = (state, items = []) => {
   if (items.length === 0) {
-    return state;
+    return;
   }
 
   const cById = { ...state.cById };
```

Comment:
not critical: as I understand you  are not need copy state object for update

Reply:
true, made a note to myself to fix it in the next PR